### PR TITLE
Refactor the way scripts are included in header.

### PR DESCRIPTION
**Pull request by dionyziz**

### CODE REVIEW

**Comment by dionyziz** [comment]
Depends #298 #299 #302

File: static/style/footer.css [hunk 1]
```diff
@@ -29,3 +29,6 @@ body {
 #main {
     margin-bottom: 65px;
 }
+#footer .dev {
```
Comment (pkakelas) [inline static/style/footer.css]: Where is dev class isset?

File: helpers/html.php [hunk 2]
```diff
@@ -221,6 +221,14 @@ function includeStyle( $path ) {
             ?>" /><?php
     }
 
+    function includeScript( $path ) {
+        ?>
+        <script type="text/javascript" src="static/script/<?php
+        echo $path;
```
Comment (vitsalis) [inline helpers/html.php]: it needs one more tab

File: helpers/html.php [hunk 3]
```diff
@@ -221,6 +221,14 @@ function includeStyle( $path ) {
             ?>" /><?php
     }
 
+    function includeScript( $path ) {
+        ?>
+        <script type="text/javascript" src="static/script/<?php
```
Comment (vitsalis) [inline helpers/html.php]: maybe put the `?>` on the begining of this line. That's how I do it at least...

**Comment by vitsalis** [comment]
LGTM

**Pull request merged**